covid19.datacommons.io/portal/gitops.json

```diff
@@ -167,63 +167,46 @@
   "analysisTools": [ ],
   "dataExplorerConfig": {
     "charts": {
-      "study_submitter_id": {
+      "project_id": {
         "chartType": "count",
-        "title": "Studies"
+        "title": "Projects"
       },
-      "virus_subtype": {
-        "chartType": "pie",
-        "title": "Virus Subtype"
+      "country_region": {
+        "chartType": "bar",
+        "title": "Country Region"
       },
-      "strain": {
-        "chartType": "pie",
-        "title": "Virus Strain"
+      "province_state": {
+        "chartType": "bar",
+        "title": "Province State"
       }
     },
     "filters": {
       "tabs": [
         {
-          "title": "Subject",
+          "title": "Location",
           "fields":[
-            "study_submitter_id",
-            "gender",
-            "race",
-            "ethnicity",
-            "vital_status"
+            "project_id",
+            "country_region",
+            "province_state",
+            "latitude",
+            "longitude"
```

Comment:
let's remove these, they are displayed as unique items instead of numbers because they are of type `string` in the dictionary